### ISSUE TITLE
[Lab2][4.4]

### ISSUE THREAD

**Issue by jimakos96** [ntua-cvsp-lab/DSP_LabSupport#87]
Στο 4.4 που ζητάει να εφαρμόσουμε το Wiener και σε σήματα  περπατήματος εμείς θα χρησιμοποιήσουμε το  θορύβο  των 400 δειγμάτων που έχουμε εξάγει από κάποιο σήμα ύπνου ? Επίσης στα σήματα ύπνου   που παράγονται από επιταχυνσιόμετρο και γυροσκόπιο  υπάρχει περίπτωση να έχουν διαφορετικής τάξης θόρυβο ,δηλ θα πρέπει να παίρνουμε διαφορετικό κομμάτι  θορύβου κάθε φορά ? Ευχαριστώ πολύ

**Comment by filby89** [comment]
Ναι, θα χρησιμοποιήσετε αυτόν τον θόρυβο που βρήκατε στο 4.3 και για τα σήματα του 4.4 εφόσον υποθέτουμε ότι ο θόρυβος αυτός είναι αντιπροσωπευτικός του σένσορα. 
Η γωνιακή ταχύτητα έχει διαφορετικές τιμές από την γραμμική επιτάχυνση (και αρκετά μεγαλύτερες) επομένως θα πρέπει για τα σήματα κάθε σένσορα να χρησιμοποιήσετε τον δικό του θόρυβο.
